sqlstore/src/main/resources/db/migration/V8812__LibraryTemplate_Multiple_Projects.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS `LibraryTemplate_Project`;
+
+CREATE TABLE `LibraryTemplate_Project` (
+  `LibraryTemplate_libraryTemplateId` bigint(20) NOT NULL,
+  `Project_projectId` bigint(20) NOT NULL,
```

Comment:
Please rename to `libraryTemplateId` and `projectId`. We've left the older fields named this way, but have been getting away from those long foreign key field names